app/presenters/documents_info_export_presenter.rb

```diff
@@ -0,0 +1,86 @@
+class DocumentsInfoExportPresenter
+  attr_reader :lead_organisation, :type
+
+  def initialize(lead_organisation:, type:)
+    @lead_organisation = lead_organisation
+    @type = type
+  end
+
+  def call
+    document_ids.sort.map do |id|
+      { document_id: id,
+        document_information: {
+          locales: locales(id),
+          subtypes: subtypes(id),
+          lead_organisations: lead_orgs(id)
+        } }
+    end
+  end
+
+  private
+
+  def document_ids
+    @document_ids ||= type.constantize
```

Comment:
Yup good point, I hadn't thought of the security implications. I've re-written the code to use `Edition` instead of the `Edition` type, it does mean however there is some sql in the joins as edition doesn't define a join with tables like organisations.